src/execution/window_segment_tree.cpp

```diff
@@ -308,8 +318,13 @@ void WindowCustomAggregator::Evaluate(WindowAggregatorState &lstate, const DataC
 	//	TODO: window should take a const Vector*
 	auto &lcstate = lstate.Cast<WindowCustomAggregatorState>();
 	auto &frames = lcstate.frames;
-	auto params = lcstate.inputs.data.data();
+	auto params = const_cast<DataChunk &>(inputs).data.data();
```

Comment:
It is because of the `window` signature but since I'm fixing `window_init` I can deal with that too.

Reply:
Turns out there is a place in `ListWindow` that assumes it can munge the partition data, but now that the API has been fixed this is glaringly obvious...